onyo/lib/utils.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+from typing import Dict, Iterable, Union
+
+from ruamel.yaml import YAML, scanner
```

Comment:
I see that this has removed the pyre ignore directive. Have you tested that it finally detects `ruamel` correctly?

Reply:
@aqw Actually not intentional - import was auto-generated and haven't run pyre on it yet. So, thx - will try!